### ISSUE TITLE
setonix-cpu profile needs rocm

### ISSUE THREAD

**Issue by psharda** [quokka-astro/quokka#563]
**Describe the bug**
We need `module load rocm/5.4.3` in `setonix-cpu.profile` otherwise setonix cannot find HIP. 

**To Reproduce**
```
source scripts/setonix-cpu.profile
mkdir build
cd build
cmake -C ../cmake/setonix.cmake .. -DCMAKE_BUILD_TYPE=Release -DQUOKKA_PYTHON=OFF -DAMReX_SPACEDIM=3 -DCMAKE_C_COMPILER=cc -DCMAKE_CXX_COMPILER=CC
```

**Additional context**
Maybe something changed in setonix which is why loading rocm for CPU only build on setonix was not needed.
Thanks to @aditivijayan for figuring it out.

**Comment by psharda** [comment]
Will be fixed by #571

**Comment by BenWibking** [comment]
I deleted the setonix-cpu profile, since Setonix GPUs seem to work now for everything except chemistry. If we need this again in the future, feel free to re-open.
